lib/node_modules/@stdlib/stats/base/dists/triangular/ctor/benchmark/benchmark.js

```diff
@@ -96,16 +107,21 @@ bench( pkg+'::set:a', function benchmark( bm ) {
 	c = 110.0;
 	dist = new Triangular( a, b, c );
 
+	len = bm.iterations;
+	y = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		y[ i ] = ( 100.0*uniform( 0.0, 1.0 ) ) + EPS;
```

Comment:
```suggestion
		y[ i ] = uniform( EPS, 100.0 );
```

Throughout the PR, the expressions involving `uniform` will be in the form of `uniform( a, b )`, where `a` and `b` represent the lower and upper bounds.